src/lightning_app/utilities/cli_helpers.py

```diff
@@ -116,7 +116,7 @@ def _retrieve_application_url_and_available_commands(app_id_or_name_or_url: Opti
                     raise Exception("The application is starting. Try in a few moments.")
                 resp = requests.get(lightningapp.status.url + "/openapi.json")
                 if resp.status_code != 200:
-                    raise Exception(f"The server didn't process the request properly. Found {resp.json()}")
+                    raise Exception("The server didn't process the request properly. Try later on.")
```

Comment:
Can we improve this message? Not sure what `Try later on` means, does the user need to wait for a few minutes/hours or they can try immediately?